ledger-core/virtual/integration/deduplication_test.go

```diff
@@ -73,7 +77,7 @@ func TestDeduplication_Constructor_DuringExecution(t *testing.T) {
 		})
 	}
 
-	typedChecker := server.PublisherMock.SetTypedChecker(ctx, t, server)
+	typedChecker := server.PublisherMock.SetTypedChecker(ctx, mc, server)
 	typedChecker.VCallResult.SetResend(false)
 	typedChecker.VDelegatedCallRequest.SetResend(true)
```

Comment:
you can ignore resend here and create a delegation token by yourself to reduce test scope.